cli/pkg/commands/images/images.go

```diff
@@ -129,6 +151,8 @@ func (m imagesModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		return m, tea.Quit
 	case modelImageSyncedMsg:
 		return m, tea.Quit
+	case modelImageDescribeMsg:
+		m.selectedImage = msg.selectedImage
```

Comment:
```suggestion
		m.selectedImage = msg.selectedImage
		return m, tea.Quit
```

You need to return a quit command to be executed after the view is displayed, Without this, the command prints the description and freezes, it never exits and you have to kill it.

Reply:
@devguyio yep true, just noticing it... have added the return. Thanks :)